air/src/air/context.rs

```diff
@@ -14,8 +14,8 @@ use utils::collections::Vec;
 pub struct AirContext<B: StarkField> {
     pub(super) options: ProofOptions,
     pub(super) trace_info: TraceInfo,
-    pub(super) main_transition_constraint_degrees: Vec<TransitionConstraintDegree>,
-    pub(super) aux_transition_constraint_degrees: Vec<TransitionConstraintDegree>,
+    pub main_transition_constraint_degrees: Vec<TransitionConstraintDegree>,
+    pub aux_transition_constraint_degrees: Vec<TransitionConstraintDegree>,
```

Comment:
Instead of making these public, I'd prefer to add a higher-level function which would merge and return these degrees. Maybe something like `pub fn get_transition_constraint_degress(&self) -> Vec<TransitionConstraintDegree>`.